api/grpcserver/smesher_service_test.go

```diff
@@ -89,6 +92,44 @@ func TestStartSmeshingPassesCorrectSmeshingOpts(t *testing.T) {
 	require.NoError(t, err)
 }
 
+func TestStartSmeshingFailsMultiSmeshing(t *testing.T) {
```

Comment:
Primarily for simplicity multi-smeshing and supervised smeshing are mutually exclusive. A node cannot use a supervised post-service if the user wants it to manage multiple identities. If that would be possible we would need to handle one ID differently from others.

It would also be a bit confusing for instance with `StartSmeshing` and `StopSmeshing`, where those GRPC calls only stop the supervised identity, but not the others (which imo it shouldn't - that's what the post-service API is for).

I renamed the test to `TestStartSmeshing_ErrorOnMultiSmeshingSetup` to make clearer what it is actually testing.

Reply:
The term "multi smeshing" is unintuitive in general IMO. There are currently 2 modes of node operation:
- supervised, in which it self-controls a local identity (initializes its POST, spawns post-service etc.),
- non-supervised, where (possibly multiple, but it could be 1) remote post-services can attach.

I don't know what a good alternative is (to use also in docs and guides to refer to this mode) :(

Reply:
Sure, we can call it supervised and remote smeshing, I don't insist on the term Multi-Smeshing. I'm just advocating against making "hybrid smeshing" a thing were a node does both.